### ISSUE TITLE
Note Block #Mine

### ISSUE THREAD

**Issue by timhow38** [cabaletta/baritone#3303]
## What do you need help with?
Is it possible to search and mine for a spesfic noteblock type?
e.g.  #mine minecraft:note_block[note=12,instrument=guitar] 
It would be really helpful with data packs that load different items as other blocks.

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by ZacSharp** [comment]
If you are on 1.12.2 can try mining for a specific state by using `minecraft:note_block:<meta>`. Meta can only be 0-15 so it shouldn't be difficult to find the value you need.

**Comment by timhow38** [comment]
> If you are on 1.12.2 can try mining for a specific state by using `minecraft:note_block:<meta>`. Meta can only be 0-15 so it shouldn't be difficult to find the value you need.

What if you are using 1.18?
Or does what you suggest work for 1.12.2 and above?

**Comment by ZacSharp** [comment]
No, the meta value doesn't exist in the post flattening versions (1.13+).
Baritone doesn't have a feature to specify blockstates for those versions.
